{
  "api": "index",
  "file": "/test/platinum/inference/47_semantic_text_knn.yml",
  "name": "Text expansion query against semantic_text field using a dense vector model returns an failure",
  "origin": "yaml",
  "request": {
    "args": {
      "body": {
        "inference_field": [
          "inference test",
          "another inference test"
        ]
      },
      "id": "doc_1",
      "index": "test-semantic-text-index-using-dense-vector",
      "refresh": true
    }
  },
  "response": {
    "headers": {
      "content-length": "559",
      "content-type": "application/json",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "error": {
        "reason": "Inference id [dense-inference-id] not found for field [inference_field]",
        "root_cause": [
          {
            "reason": "Inference id [dense-inference-id] not found for field [inference_field]",
            "suppressed": [
              {
                "reason": "Inference id [dense-inference-id] not found for field [inference_field]",
                "type": "resource_not_found_exception"
              }
            ],
            "type": "resource_not_found_exception"
          }
        ],
        "suppressed": [
          {
            "reason": "Inference id [dense-inference-id] not found for field [inference_field]",
            "type": "resource_not_found_exception"
          }
        ],
        "type": "resource_not_found_exception"
      },
      "status": 404
    },
    "statusCode": 404
  }
}